source/XMLFormatting.py

```diff
@@ -48,10 +55,12 @@ def _EndElementHandler(self,tagName):
 		else:
 			raise ValueError("unknown tag name: %s"%tagName)
 
-	def _CharacterDataHandler(self,data):
+	def _CharacterDataHandler(self,data, processBufferedSurrogates=False):

```

Comment:
Why was the `processBufferedSurrogates` arg added?

Reply:
Ah, I added this parameter without actually using it. Let me fix that.